data/remnant missions.txt

```diff
@@ -655,149 +655,288 @@ mission "Remnant: Learn Sign Follow Up"
+			label the_rest
+			`	You lead her back to your ship, where you show her the plasma cannon. "I have never seen weapons designed to handle such a high heat output! Is this some kind of plasma based weapon?" she asks, gesturing at a cannon. You nod, and she continues, "Intriguing. We have noticed that the Korath ships seem to run particularly hot, and have speculated that overheating them might be an effective means of disabling them. I would enjoy researching these a bit more. If you could deliver two to <planet> for us you would be well compensated."`
+				accept
+	on complete
+		outfit "Plasma Cannon" -2
+		payment 1130000
+		"remnant met taely" ++
+		conversation
+			`As the <ship> settles onto the pad closest to the research facility, you can see a weapons specialist checking over a large cannon on a platform beneath a nearby Albatross. When he sees you starting to unload the two plasma cannons, he snaps the casing closed and gestures to someone out of sight. Within moments both him and an empty flatbed are there to pick up the weapons.`
+			`	"Thank you for your help," he says. "Our Inhibitors provide us with something of a mobility advantage against the Korath, but they aren't particularly effective at disabling them. These plasma cannons will open new avenues of research for us." He pauses, then continues. "I also included a bonus for you. It is a 'finder's fee,' as we always appreciate new tech."`
+			branch wave
+				has "Remnant: Learn Sign 1: done"
+			`	The weapons technician finishes strapping down the plasma cannons and makes a gesture towards you, then heads off toward the lab.`
+			goto end
+			label wave
+			`	The weapons technician finishes strapping down the plasma cannons and makes a gesture that you recognize as "farewell," then heads off toward the lab.`
+			label end
+			`	As you watch the remnant heading off with their new weapons to study, you can not help but wonder what will be the result of it.`
 
-			`	"I'll be careful," she sings with a tinge of sadness. "I am an engineer, not a Korath."`
-				goto more
 
-			label deep
-			`	"Fascinating," she sings as she clambers over the <ship> with the grace of someone who obviously knows her ships. "Above all else, I am an engineer of movement," she chants. "We have done well to hide as deep in the Ember Waste as we have, but our ships' limited ranges struggle to take us farther. Examining systems from elsewhere may inspire new developments."`
 
-			label more
-			`	After examining the <ship> for quite some time, she stops beneath it and looks up. "Is this a new form of ramscoop?" she asks as she gestures at your ship's ramscoop. As you nod in the affirmative, she continues. "Fuel has always been a great limiter of our exploration. If you could get a couple extra that we could dismantle, this could be very useful to us."`
+mission "Remnant: Catalytic Ramscoop"
+	name "Retrieve Catalytic Ramscoops for the Remnant"
```

Comment:
Name seems dangerously long.

Reply:
Removed the `for the Remnant` part in order to shorten it up. (the player can probably remember who they are for, and if not, that's what the description is for, right?)

Given that this same logic applies to all of these missions, I removed the same `for the Remnant` part from all six of them.